### PR TITLE
{tools}[fosscuda/2020b] udocker v1.3.4

### PR DESCRIPTION
**Pull request by gpourtois**
Test to upgrade udocker to version 1.3.4 with cuda support

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Review by casparvl — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
close / open to see if that triggers the CI to run

File: easybuild/easyconfigs/u/udocker/udocker-1.3.4-fosscuda-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'udocker'
+version = '1.3.4'
+versionsuffix = '-Python-%(pyver)s'
```
Comment (casparvl) [inline easybuild/easyconfigs/u/udocker/udocker-1.3.4-fosscuda-2020b.eb:5]: Please remove this (this is one of the things the CI is complaining about). Nowadays, we assume Python-3.X to be standard, so we don't add a version suffix for it anymore. We'll only add a versionsuffix if something still depends on Python-2.X.

File: easybuild/easyconfigs/u/udocker/udocker-1.3.4-fosscuda-2020b-Python-3.8.6.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
```
Comment (casparvl) [inline easybuild/easyconfigs/u/udocker/udocker-1.3.4-fosscuda-2020b-Python-3.8.6.eb:1]: This file can be removed alltogether, see the instructions at https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches on how to delete easyconfig files if you use EasyBuild's github integration (if you don't use the github integration, you can simply `git rm` this file from your local fork, push that change to your local fork, and this PR should update itself automatically).

**Review by casparvl — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
closing this since `fosscuda/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**